src/main/java/Event.java

```diff
@@ -0,0 +1,18 @@
+public class Event extends Task{
```

Comment:
Shouldn't `Task` and the proceeding `{` afterwards be separated by a single space?

Reply:
Thanks for pointing it out!